.env.template

```diff
@@ -0,0 +1,18 @@
+# private strings
+REACT_APP_PORTIS_DAPP_ID=006cdb48-470e-466c-aa20-c4de6ebd10e7
```

Comment:
Are these actual secrets, or are they placeholder vars?

Reply:
These values are placeholders made to look like real values.